lms/static/js/instructor_dashboard/membership.js

```diff
@@ -7,16 +7,18 @@ imports from other modules.
 wrap in (-> ... apply) to defer evaluation
 such that the value can be defined later than this assignment (file load order).
 */
-
+var Membership;
```

Comment:
Is this var necessary? I ask because I would think the code below puts Membership on the window object.